solr/prometheus-exporter/src/test/org/apache/solr/prometheus/scraper/SolrStandaloneScraperTest.java

```diff
@@ -47,42 +54,45 @@ public class SolrStandaloneScraperTest extends RestTestBase {
 
   @BeforeClass
   public static void setupBeforeClass() throws Exception {
-    File tmpSolrHome = createTempDir().toFile();
-    tmpSolrHome.deleteOnExit();
-
-    FileUtils.copyDirectory(new File(TEST_HOME()), tmpSolrHome.getAbsoluteFile());
-
-    initCore("solrconfig.xml", "managed-schema");
+    solrRule.startSolr(LuceneTestCase.createTempDir());
 
-    createJettyAndHarness(
-        tmpSolrHome.getAbsolutePath(), "solrconfig.xml", "managed-schema", "/solr", true, null);
+    Path configSet = LuceneTestCase.createTempDir();
+    createConf(configSet);
+    solrRule.newCollection().withConfigSet(configSet.toString()).create();
 
+    PrometheusExporterSettings settings = PrometheusExporterSettings.builder().build();
+    SolrScrapeConfiguration scrapeConfiguration =
+        SolrScrapeConfiguration.standalone(solrRule.getBaseUrl());
+    solrClient =
+        new SolrClientFactory(settings, scrapeConfiguration)
+            .createStandaloneSolrClient(solrRule.getBaseUrl());
     executor =
         ExecutorUtil.newMDCAwareFixedThreadPool(
             25, new SolrNamedThreadFactory("solr-cloud-scraper-tests"));
     configuration =
         Helpers.loadConfiguration("conf/prometheus-solr-exporter-scraper-test-config.xml");
-
-    solrClient =
-        new Http2SolrClient.Builder(restTestHarness.getAdminURL())
-            .withResponseParser(new NoOpResponseParser("json"))
-            .build();
     solrScraper = new SolrStandaloneScraper(solrClient, executor, "test");
 
     Helpers.indexAllDocs(solrClient);
   }
 
+  public static void createConf(Path configSet) throws IOException {
+    Path subHome = configSet.resolve("conf");
+    Files.createDirectories(subHome);
+
+    Path top = SolrTestCaseJ4.TEST_PATH().resolve("collection1").resolve("conf");
+    Files.copy(top.resolve("managed-schema.xml"), subHome.resolve("schema.xml"));
+    Files.copy(top.resolve("solrconfig.xml"), subHome.resolve("solrconfig.xml"));
+
+    Files.copy(top.resolve("stopwords.txt"), subHome.resolve("stopwords.txt"));
```

Comment:
wish we had a different more minimal config that dind't drag along stopwords and synnymos....

Reply:
trouble with using the minimal config is that the data indexed here uses multi value strings heavily which is not allowed in the existing min config. it will fail indexing, so this is the best I could come up with

Reply:
makes sense.  Maybe someday we figure out how many tests actually want stopwords, or which ones want stopwords, and only have those tests depend on it!